sources/shared-defaults/defaults.toml

```diff
@@ -167,6 +167,18 @@ affected-services = ["lockdown"]
 path = "/etc/corndog.toml"
 template-path = "/usr/share/templates/corndog-toml"
 
+# Bootstrap Commands
+[services.bootstrap-commands]
+configuration-files = ["bootstrap-commands-toml"]
+restart-commands = []
+
+[metadata.settings.bootstrap-commands]
+affected-services = ["bootstrap-commands"]
```

Comment:
You will need to add a separate migration that uses `AddMetadataMigration` to get rid of this newly-added metadata on downgrade.

Reply:
Added and tested new migrations